packages/coinstac-ui/app/render/components/field-password.jsx

```diff
@@ -20,8 +20,8 @@ export default class FieldPassword extends Component {
   }
 
   handleChange() {
-    const password = findDOMNode(this.refs.password).value;
-    const confirmPassword = findDOMNode(this.refs.confirm).value;
+    const password = findDOMNode(this.formPassword).value;
```

Comment:
I guess React doesn't like it anymore: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md